BlueprintUI/Tests/LayoutWriterTests.swift

```diff
@@ -34,6 +34,42 @@ class LayoutWriterTests : XCTestCase {
         XCTAssertEqual(buildCount, 2)
     }
     
+    func test_zeroSize_does_no_layout() {
+        
+        // Trying to perform a layout with a zero size doesn't really mean anything.
```

Comment:
FWIW there are some cases where layout with a zero size is still valid:
- a spacer that is zero in 1 dimension
- children that overflow
- children that are anchored but do not affect layout, e.g. a shadow or a badge